docs/source/index.rst

```diff
@@ -49,32 +49,32 @@ features:
 * Integrated access to the pdb debugger and the Python profiler.
 
 
-The Command line interface inherits all the above functionality and posses 
+The Command line interface inherits all of the above functionalities and possess 
```

Comment:
I would agree with `functionality` and swap out `possess` for `adds`. `...inherits the above functionality and adds...`